php_phongo.c

```diff
@@ -56,27 +56,28 @@ zend_object_handlers* phongo_get_std_object_handlers(void)
 /* }}} */
 
 /* {{{ Memory allocation wrappers */
-static void* php_phongo_malloc(size_t num_bytes) /* {{{ */
```

Comment:
I thought I had removed all of these, but I guess I haven't. Thanks for cleaning this up!

Reply:
Is "these" referring to the foldmarkers (i.e. `{{{` and `}}}`)? There are a bunch more, so I can go through and clean those up separately if we like. None of us are using vim to work on this project, so I don't have to worry about @bjori yelling at me.